### PR TITLE
SAJ integration doc update with UI configuration

### PR DESCRIPTION
**Pull request by sanyatuning**
## Proposed change

Reopened PR #15885
SAJ integration doc update with UI configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46853
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/saj.markdown [hunk 1]
```diff
@@ -21,42 +23,15 @@ The Wi-Fi module requires a username and password for authentication where the e
 
 ## Configuration
 
-To enable this sensor, add the following lines to your `configuration.yaml` file:
+1. In the **Configuration** > **Integrations** menu, click the `+ ADD INTEGRATION` button on the lower-right of the screen.
+2. Select **SAJ Solar Inverter** from the pop-up menu.
+3. Complete the dialog box and click `SUBMIT`.
```
Comment (frenck) [inline source/_integrations/saj.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

Please note: `## Configuration` also needs to be removed in this case. (Cannot suggest that, as it is outside of the change set).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/saj.markdown (hunk 1 above)
Comment (sanyatuning) [reply source/_integrations/saj.markdown]: Done, thanks

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**